docs/dev-container.md

```diff
@@ -0,0 +1,57 @@
+# Dev Container
+
+## Building the Dev Container
+
+> _**WARNING:** This step is critical to ensure you correctly build the Dev
+Container image._
```

Comment:
Does this format correctly, or does it need a leading `> `?

Reply:
I was discussing line 6. Does it ALSO need a `> `? I don't know the answer. Is it rendering correctly.